examples/custom-transaction-validation/modules/l1/src/main/scala/com/my/custom_validator/l1/Main.scala

```diff
@@ -0,0 +1,32 @@
+package com.my.custom_validator.l1
+
+import java.util.UUID
+import org.tessellation.BuildInfo
+import org.tessellation.currency.l1.CurrencyL1App
+import org.tessellation.schema.cluster.ClusterId
+import org.tessellation.schema.transaction.Transaction
+import org.tessellation.schema.semver.MetagraphVersion
+import org.tessellation.schema.semver.TessellationVersion
+import org.tessellation.security.Hashed
+import org.tessellation.dag.l1.domain.transaction.CustomContextualTransactionValidator
+import org.tessellation.dag.l1.domain.transaction.ContextualTransactionValidator.CustomValidationError
+import org.tessellation.dag.l1.domain.transaction.TransactionValidatorContext
+import eu.timepit.refined.auto._
+
+object Main
+  extends CurrencyL1App(
+    "custom-transaction-validation-l1",
+    "custom-transaction-validation L1 node",
+    ClusterId(UUID.fromString("517c3a05-9219-471b-a54c-21b7d72f4ae5")),
+    tessellationVersion = TessellationVersion.unsafeFrom(BuildInfo.version),
+    metagraphVersion = MetagraphVersion.unsafeFrom(BuildInfo.version)
+  ) {
+  override def transactionValidator: Option[CustomContextualTransactionValidator] = Some {
+    (hashedTransaction: Hashed[Transaction], context: TransactionValidatorContext) =>
+      Either.cond(
+        context.currentOrdinal.value < 10L || hashedTransaction.fee.value % 2 == 0,
```

Comment:
I think we should make this a more realistic example. I can't think of a real use case for a fee that must be an even number. How about we replace the even number requirement with a minimum fee?

Reply:
Sure, I will replace it then :+1: